apps/src/StudioApp.js

```diff
@@ -2003,7 +2003,7 @@ StudioApp.prototype.handleHideSource_ = function (options) {
         }
       }
 
-      if (!options.embed && !options.noHowItWorks) {
+      if (!options.embed && (!options.noHowItWorks || this.config.droplet === false)) {
```

Comment:
Nicely done identifying how to show the button for SpriteLab! I'm wondering if this code will ever get us into a situation where we initialize `options.noHowItWorks = true` (so we expect not to see the 'How It Works' button) in another blockly lab, but since we're not in droplet, we see the button even though we specified otherwise. What if we move the `this.config.droplet === false` check to where `config.noHowItWorks` is set in GameLab.js (https://github.com/code-dot-org/code-dot-org/blob/f9513b213e952d36bf4a90cd5c810cc7d0871a85/apps/src/gamelab/GameLab.js#L268) and use it to set the value based on whether we are in GameLab or SpriteLab? I think that will scope down the change to affect just GameLab and SpriteLab.

Reply:
Congrats on the first PR! Great work!! Just gonna add in another piece of info. This appears to be how we determine whether Game Lab is in Sprite Lab mode: https://github.com/code-dot-org/code-dot-org/blob/cee015150a70fb3b734895e348deea325bc4073f/apps/src/gamelab/GameLab.js#L194

Reply:
I see! I moved the `this.config.droplet === false` to GameLab.js in line 268 and now the `config.noHowItWorks` is passed in as false in StudioApp.js. noHowItWorks seems to be passed in as false without having to add more logic in line 194 in GameLab.js so I just left it as is unless I'm missing something which is highly likely. I pushed the changes to this pull request.